### DISCUSSION TITLE
Git-Sim Installation (standalone package)

### DISCUSSION THREAD

**Discussion by initialcommit-io** [initialcommit-com/git-sim#67] [General]
@paketb0te @abhijitnathwani 

Git-Sim currently requires multiple steps for installation, including installing Manim.

It would be very nice for users if Git-Sim could be installed as standalone package on Windows, MacOS and Linux.

I don't have much experience packaging applications in that way, so starting this discussion to get input and direction on this.

**Comment by buhtz** [comment]
See #85 

When this is "implemented" then you just have to do `python3 -m pip install .` (if you need including `--editable` argument).
That same command is used by nearly every modern GNU/Linux distro (including Debian) to create distro specific packages of a python project.
